plutus-benchmark/common/PlutusBenchmark/NaturalSort.hs

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE DerivingStrategies #-}
```

Comment:
It's enabled globally in `plutus-benchmark.cabal`, so it shouldn't be needed here.

Reply:
@perturbing Did you have some reason for adding that, like you needed it to use a copy of the code outside our repository?  I've removed this and another one that don't seem to be needed.